docs/UserGuide.md

```diff
@@ -11,117 +11,213 @@
    > Having any Java 8 version is not enough. <br>
    This app will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the 'releases' tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `MustDoList.jar` from the 'releases' tab.
+2. Copy the file to the folder you want to use as the home folder for your MustDoList.
 3. Double-click the file to start the app. The GUI should appear in a few seconds. 
    > <img src="images/Ui.png" width="600">
 
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
+   * **`list`** : lists all task
+   * **`add`**` Submission of CS2103 Project ed/11112016 et/0900hrs` : 
+     adds a task named `Submission of CS2103 Project` to the MustDoList.
+   * **`delete`**` 1` : deletes the 1st task shown in the current list
    * **`exit`** : exits the app
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
 ## Features
 
-#### Viewing help : `help`
+####6.1 Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
- 
-> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
-> items with `...` after them can have multiple instances. Order of parameters are fixed. 
-> 
-> Persons can have any number of tags (including 0)
+####6.2.1 Adding a task: `add`
+Adds a task to the MustDoList.<br>
 
-Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+Format1: `add TASK_NAME`
+Format2: `add TASK_NAME by ed/END_DATE`
+Format3: `add TASK_NAME by [sd/START_DATE] ed/END_DATE [st/START_TIME] et/END_TIME`
```

Comment:
I think `by` is not used here, so you can remove it. Also, you can consider making other parameters optional, as I see no harm in doing so.